analyze_openapi_changes.py

```
@@ -0,0 +1,114 @@
+    except Exception as e:
+        print(f"Error loading {file_path}: {e}")
+        return None
+
+def analyze_changes(old_schema: Dict[str, Any], new_schema: Dict[str, Any]) -> Dict[str, Any]:
+    diff = DeepDiff(old_schema, new_schema, view='tree')
+
+    api_changes = {
+        "new_endpoints": [],
+        "removed_endpoints": [],
+        "modified_endpoints": [],
+        "security_changes": []
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)